processor/src/main/java/com/epam/learn/processor/service/ResourceService.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.learn.processor.service;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.core.io.ByteArrayResource;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+@FeignClient(value = "resource-service", url = "http://localhost:8091")
+public interface ResourceService {
+
+    @GetMapping("/resources/{id}")
+    ByteArrayResource getById(@PathVariable(value = "id") Integer resourceId);
```

Comment:
I believe you can use more general interface instead of `ByteArrayResource`:
`org.springframework.core.io.Resource`